examples/sphere/solidbody.jl

```diff
@@ -107,10 +66,131 @@ function linkfig(figpath, alt = "")
+            h0 * exp(-(rd / r0)^2 / 2)
+        else # default test case, cosine bell
+            if rd < r0
+                h0 / 2 * (1 + cospi(rd / r0))
+            else
+                0.0
+            end
+        end
+    end
+
+    u = map(coords) do coord
+        ϕ = coord.lat
+        λ = coord.long
 
-#----------------lat, long plots
-Plots.png(Plots.plot(coords.lat), joinpath(path, "latitude.png"))
-linkfig("output/$(dirname)/latitude.png", "latitude")
-Plots.png(Plots.plot(coords.long), joinpath(path, "longitude.png"))
-linkfig("output/$(dirname)/longitude.png", "longitude")
+        uu = u0 * (cosd(α0) * cosd(ϕ) + sind(α0) * cosd(λ) * sind(ϕ))
+        uv = -u0 * sind(α0) * sind(λ)
+        Geometry.UVVector(uu, uv)
+    end
+
+    function rhs!(dh, h, u, t)
+        div = Operators.Divergence()
+
+        @. dh = -div(h * u) # strong form of equation
+        Spaces.weighted_dss!(dh)
+    end
+
+    # Set the RHS function
+    rhs!(similar(h_init), h_init, u, 0.0)
+
+    # Solve the ODE
+    T = 86400 * 12
+    dt = 20 * 60
+    prob = ODEProblem(rhs!, h_init, (0.0, T), u)
+    sol = solve(
+        prob,
+        SSPRK33(),
+        dt = dt,
+        saveat = dt,
+        progress = true,
+        adaptive = false,
+        progress_message = (dt, u, p, t) -> t,
+    )
+    L1err[k] = norm(sol.u[end] .- h_init, 1)
+    L2err[k] = norm(sol.u[end] .- h_init)
+    Linferr[k] = norm(sol.u[end] .- h_init, Inf)
+
+    @info "Test case: $(test_name) with α: $(test_angle_name)"
+    @info "Number of elements per cube panel: $(ne) x $(ne)"
+    @info "Solution norm at t = 0: ", norm(h_init)
+    @info "Solution norm at t = $(T): ", norm(sol.u[end])
+    @info "L₁ error at t = $(T): ", L1err[k]
+    @info "L₂ error at t = $(T): ", L2err[k]
+    @info "L∞ error at t = $(T): ", Linferr[k]
+end
+
+# Print convergence rate info
+conv = convergence_rate(L2err, Δh)
+@info "Converge rates for this test case are: ", conv
+
+# Plot the errors
+# L₁ error Vs number of elements
+Plots.png(
+    Plots.plot(
+        collect(ne_seq),
+        L1err,
+        yscale = :log10,
+        xlabel = "Nₑ",
+        ylabel = "log₁₀(L₁ err)",
+        label = "",
+    ),
+    joinpath(path, "L1error.png"),
+)
+linkfig(relpath(joinpath(path, "L1error.png"), "../.."), "L₁ error Vs Nₑ")
+
+# L₂ error Vs number of elements
+Plots.png(
+    Plots.plot(
+        collect(ne_seq),
+        L2err,
+        yscale = :log10,
+        xlabel = "Nₑ",
+        ylabel = "log₁₀(L₂ err)",
+        label = "",
+    ),
+    joinpath(path, "L2error.png"),
+)
+linkfig(relpath(joinpath(path, "L2error.png"), "../.."), "L₂ error Vs Nₑ")
+
+# L∞ error Vs number of elements
+Plots.png(
+    Plots.plot(
+        collect(ne_seq),
+        Linferr,
+        yscale = :log10,
+        xlabel = "Nₑ",
+        ylabel = "log₁₀(L∞ err)",
+        label = "",
+    ),
+    joinpath(path, "Linferror.png"),
+)
+linkfig(relpath(joinpath(path, "Linferror.png"), "../.."), "L∞ error Vs Nₑ")
```

Comment:
I had this wrong: it should be
```suggestion
linkfig(relpath(joinpath(path, "Linferror.png"), joinpath(@__DIR__, "../..")), "L∞ error Vs Nₑ")
```

Reply:
and above as well